menu_tools/object_performance/turnon_collection.py

```diff
@@ -24,7 +24,13 @@ def _transform_key(self, raw_key: str, obj: str):
         NTuple branch names for quality and region
         to "quality"/"region".
         """
-        key = raw_key.removeprefix(obj).lower()
+        ## nano
+        if ("_" in raw_key) and ("dr_0" not in raw_key):
```

Comment:
In theory this could be `dr_1` as well https://github.com/cms-l1-dpg/Phase2-L1MenuTools/blob/0ca5cf27473788010c3186ab42384382ded5d994/menu_tools/caching/cache_objects.py#L25C9-L25C64